docs/adr/0002-use-react-redux-and-typescript-for-subscription-management-pages.md

```diff
@@ -0,0 +1,129 @@
+# Use React, Redux, and Typescript for subscription management pages
+
+* Deciders: Ben Bangert, Ian Bicking, Les Orchard
+* Date: 2019-05-08
+
+## Context and Problem Statement
+
+A solution for isolating third-party payment widgets from the rest of account
+management can include building a separate web app on its own dedicated domain.
+As a side effect in building that app, we have the opportunity to choose
+technologies for building it that don't necessarily follow the rest of FxA.
+
+## Decision Drivers
+
+* Opportunity for a fresh start with tech stack without rewriting
+* Security in dealing with payment transactions
```

Comment:
I'd like to read more about this.

Reply:
It's a decision driver - but I think in terms of pros & cons, it's a wash. 

* Backbone & associated dependencies are aging, yet I can't say that the infrequent updates should have presented any significant security issues. If there were, I expect FxA as a project would have long since mitigated them.

* React & friends are frequently updated and also have a good security story, IMO. React makes it very hard & obvious to do things like [render unescaped HTML into the DOM](https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml). That doesn't seem to be a thing Backbone helps or hinders except through Mustache in the initial templates and just avoiding innerHTML shenanigans in View code.